packages/contracts/test/utils/combinatorial_sets.ts

```diff
@@ -0,0 +1,25 @@
+import { BigNumber } from '@0xproject/utils';
+import * as _ from 'lodash';
+
+export const BigNumbers = [];
+
+const MAX_256_BIT_NUMBER = new BigNumber(2).pow(256).minus(1);
+
+export const positiveNaturalBigNumbers = [
```

Comment:
These values are the basis for generating the inputs that make up the different test cases.